src/encoded/static/components/regulome_search.js

```diff
@@ -690,8 +912,11 @@ class RegulomeSearch extends React.Component {
                                 >
                                     <h4>QTL Data</h4>
                                     {(this.state.selectedThumbnail === null) ?
-                                        <div className="line"><i className="icon icon-chevron-circle-right" />Click to see dsQTL and eQTL data.
-                                            <div>(<b>{QTLData.length}</b> result{QTLData.length !== 1 ? 's' : ''})</div>
+                                        <div>
+                                            <div className="line"><i className="icon icon-chevron-circle-right" />Click to see dsQTL and eQTL data.
+                                                <div>(<b>{QTLData.length}</b> result{QTLData.length !== 1 ? 's' : ''})</div>
+                                            </div>
+                                            <QTLDetails {...this.props} />
```

Comment:
Looks like `QTLDetails` only takes one property: context. Sometimes passing a spread makes it tough to hunt down what the called component actually requires.